docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -123,9 +123,14 @@ endif::[]
 * *Cloud region*: Select `US East, N. Virginia`.
 * *Availability zones*: Select `Multi`.
 ////
-. Use the slider to select the required instance size. Instances with more streaming units have more capacity. In {product-kafka}, you can create Kafka instances of 1 or 2 streaming units, however, instances larger than 1 streaming unit are currently available only as a Technology Preview and should not be used in production.
-. Click *Create instance* to start the creation process for your Kafka instance.
+. If you have purchased a subscription for {product-kafka}, you can use the slider to select an instance size. Instances with more streaming units have more capacity.
```

Comment:
Use a contraction here to stay consistent  with the informal voice elsewhere in the guide. "If you've purchased..."